backend/modules/pdf_processing/router.py

```
@@ -0,0 +1,287 @@
+                    return {
+                        "status": "error",
+                        "message": "Invalid API key - authentication failed",
+                        "details": "Please check your LLAMAPARSE_API_KEY"
+                    }
+                else:
+                    return {
+                        "status": "error",
+                        "message": f"API request failed (status: {e.response.status_code})",
+                        "details": str(e)
+                    }
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)